src/main/java/ca/mcgill/mcb/pcingola/snpEffect/commandLine/SnpEffCmdEff.java

```diff
@@ -629,17 +644,29 @@ else if (outFor.equals("GATK")) {
 						} else if (outFor.equals("TXT")) usage("Output format 'TXT' has been deprecated. Please use 'VCF' instead.\nYou can extract VCF fields to a TXT file using 'SnpSift extractFields' (http://snpeff.sourceforge.net/SnpSift.html#Extract).");
```

Comment:
ok. -htmlStats seems redundant then.  What do you think?  I'll make createSummaryHTML default to TRUE and other appropriate changes.